guide/src/python_from_rust.md

```diff
@@ -236,6 +239,100 @@ def leaky_relu(x, slope=0.01):
 # }
 ```
 
+### Include multiple Python files
+
+You can include a file at compile time by using
+[`std::include_str`](https://doc.rust-lang.org/std/macro.include_str.html) macro.
+
+Or you can load a file at runtime by using
+[`std::fs::read_to_string`](https://doc.rust-lang.org/std/fs/fn.read_to_string.html) function.
+
+Many Python files can be included and loaded as modules. If one file depends on
+another you must preserve correct order while declaring `PyModule`.
+
+Example directory structure:
+```
```

Comment:
These files are `include_str!`'d at the bottom of our lib.rs, so unless you tell it otherwise, rustdoc is going to try to run this as a Rust code example.

```suggestion
```text
```